rib/rib_test.go

```diff
@@ -3540,7 +3544,9 @@ func TestFlush(t *testing.T) {
 
 			return r
 		}(),
-	}}
+	},
+	// TODO: xw-g add a test case to test non-zero reference.
```

Comment:
Suggest we should probably do this before we submit the change here? But how is this different from `cannot delete referenced NH` above?